storage/runtime/src/main/java/io/quarkiverse/googlecloudservices/storage/runtime/StorageConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkiverse.googlecloudservices.storage.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "google.cloud.storage", phase = ConfigPhase.RUN_TIME)
+public class StorageConfiguration {
+    /**
+     * Overrides the default service host.
```

Comment:
Maybe add something to say that it's purpose if for testing only ?

Reply:
I followed the description here; https://googleapis.dev/java/google-cloud-clients/0.97.0-alpha/com/google/cloud/ServiceOptions.Builder.html#setHost-java.lang.String-

but a remark that it is commonly used for Testing doesn’t hurt.